Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.py

```diff
@@ -375,9 +415,47 @@ def create_incident_labels(self, parsed_msg, headers):
 
         return labels
 
-    def mail_to_incident(self, msg, service, user_key):
-        parsed_msg, headers, _ = self.get_email_context(msg, user_key)
+    @staticmethod
+    def get_date_isoformat_server(dt: datetime) -> str:
+        """Get the  datetime str in the format a server can parse. UTC basded with Z at the end
 
+        Args:
+            dt (datetime): datetime
+
+        Returns:
+            str: string representation
+        """
+        return datetime.fromtimestamp(dt.timestamp()).isoformat(timespec='seconds') + 'Z'
```

Comment:
```suggestion
        return dt.replace(tzinfo=None).isoformat(timespec='seconds') + 'Z'
```

Reply:
Not exactly. Doing replace will simply drop the tz info. We want to move the date to utc. If tzino is non gmt then: `datetime.fromtimestamp(d.timestamp()) != d.replace(tzinfo=None)`